src/main/java/crystal/MainWindow.java

```diff
@@ -0,0 +1,56 @@
+package crystal;
+
+import javafx.fxml.FXML;
+
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
```

Comment:
I like how you have divided the UI into classes too , maybe putting them in folders as well would make you're directory cleaner .